service/src/main/java/bio/terra/workspace/service/resource/referenced/ReferencedResourceService.java

```diff
@@ -123,4 +129,13 @@ public ReferencedResource getReferenceResourceByName(
         userReq, workspaceId, SamConstants.SAM_WORKSPACE_READ_ACTION);
     return resourceDao.enumerateReferences(workspaceId, offset, limit);
   }
+
+  public boolean validateReference(
```

Comment:
nit: Do you need "Reference" in the method name, since it's already in the class name?

Reply:
renamed without `reference` in the name